authorization/role/repository/role_mapping_blackbox_test.go

```diff
@@ -98,7 +98,7 @@ func (s *roleMappingBlackBoxTest) createTestRoleMapping(fromResourceTypeName str
 }
 
 func (s *roleMappingBlackBoxTest) TestOKToLoad() {
-	rm1, err := s.createTestRoleMapping(authorization.IdentityResourceTypeOrganization, "employee", authorization.IdentityResourceTypeTeam, "member")
+	rm1, err := s.createTestRoleMapping(authorization.IdentityResourceTypeOrganization, "role_mapping_test_role_personnel", authorization.IdentityResourceTypeTeam, "role_mapping_test_role_member")
```

Comment:
Could you use `uuid.NewV4().String()` to generate the random strings? That will ensure you will never have a name conflict.

Reply:
There isn't actually a name conflict here.

Reply:
I should clarify - at first I thought there _was_ a name conflict, but it turned out that there wasn't. ;)

Reply:
so, is this PR only to support removal of the parent resource link?